libs/search-widget/src/common/label/Label.svelte

```diff
@@ -1,27 +1,50 @@
 <script lang="ts">
   import { createEventDispatcher } from 'svelte';
-  import { getCDN } from '../../core/utils';
   import type { Classification } from '@nuclia/core';
   import { map } from 'rxjs';
   import { labelSets } from '../../core/stores/labels.store';
+  import Icon from '../icons/Icon.svelte';
+  import { tap } from 'rxjs/operators';
 
   export let label: Classification;
+  export let clickable = false;
   export let removable = false;
 
+  let setColor = true;
   const color = labelSets.pipe(
     map((labelSet) => labelSet[label.labelset]?.color),
+    tap((color: string) => {
+      if (color === '#CCCED6') {
+        setColor = false;
+      }
+      return color;
+    })
   );
 
+
   const dispatch = createEventDispatcher();
   const remove = () => {
     dispatch('remove');
   };
+  const onClick = (event: MouseEvent | KeyboardEvent) => {
+    if (clickable) {
+      event.preventDefault();
+      event.stopPropagation();
+      dispatch('selected');
```

Comment:
any reason why you prefer `selected` over `click`?

Reply:
I thought using click would not allow me to trigger the click even only when clickable was enabled